templates/blog/singular-category.html

```diff
@@ -0,0 +1,17 @@
+{% if "display_category" in article and article.display_category.name %}
```

Comment:
Do we have single category pages? I don't see any links in the category

![category-links](https://user-images.githubusercontent.com/14939793/176162537-9e56f6a7-d87e-4da8-9028-07485f5bd83f.png)

So, should we either remove this `singular-category.html` or we put links there?